src/elements/Text.js

```diff
@@ -237,13 +237,14 @@ class Text extends Base {
       left + this.padding.left,
       top + this.padding.top - initialY,
     );
-
     // Perform actual text rendering on document
     PDFRenderer.render(this.root.instance, [this.lines]);
+    setLink(this);
```

Comment:
I have some doubts about how this is handled here, since it's textkit (via `PDFRenderer`) which renders the links. This means go-to links won't work for inline texts. But we can add that in a separate iteration on `v2.0`